app/services/send_application_to_provider.rb

```diff
@@ -0,0 +1,26 @@
+# This worker will be scheduled to run nightly
+class SendApplicationToProvider
+  attr_accessor :application_choice
+
+  def initialize(application_choice:)
+    self.application_choice = application_choice
+  end
+
+  def call
+    set_reject_by_default
+    ApplicationStateChange.new(application_choice).send_to_provider!
```

Comment:
Are we relying on the state change to persist the attribute changes? Might be better to explicitly save here, even if it costs an extra query.